shm/inc/island.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include "coordinates.hpp"
+
+class Island {
+public:
+    Coordinates getPosition() { return position_; }

```

Comment:
```suggestion
    Coordinates getPosition() const { return position_; }

```
getters's purpose is to retrieve data and they should not modify anything in the process.